sdk/monitor/azure-monitor-query/metrics-query-migration-guide.md

```diff
@@ -0,0 +1,322 @@
+Duration-based `MetricsQueryTimeInterval` is not supported in the `azure-monitor-query-metrics`.
+
+### Instantiate metrics batch clients
+
+In `azure-monitor-query`, metrics batch functionality was accessed through the `MetricsClient` and `MetricsAsyncClient` classes:
+
+```java
+import com.azure.monitor.query.MetricsClient;
+import com.azure.monitor.query.MetricsClientBuilder;
+import com.azure.monitor.query.MetricsAsyncClient;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.core.credential.TokenCredential;
+
+public class MetricsClientSample {
+    public static void main(String[] args) {
+        TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+        // Create synchronous client
+        MetricsClient metricsClient = new MetricsClientBuilder()
+                .credential(tokenCredential)
+                .buildClient();
+
+        // Create asynchronous client
+        MetricsAsyncClient metricsAsyncClient = new MetricsClientBuilder()
+                .credential(tokenCredential)
+                .buildAsyncClient();
+        
+        System.out.println("Successfully created metrics clients");
+    }
+}
+```
+
+In `azure-monitor-query-metrics`, the class names remain the same but are in a different package:
+
+```java
+import com.azure.monitor.query.metrics.MetricsClient;
+import com.azure.monitor.query.metrics.MetricsClientBuilder;
+import com.azure.monitor.query.metrics.MetricsAsyncClient;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.core.credential.TokenCredential;
+
+public class MetricsQueryMetricsClientSample {
+    public static void main(String[] args) {
+        TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+        // Create synchronous client
+        MetricsClient metricsClient = new MetricsClientBuilder()
+                .credential(tokenCredential)
+                .buildClient();
+
+        // Create asynchronous client
+        MetricsAsyncClient metricsAsyncClient = new MetricsClientBuilder()
+                .credential(tokenCredential)
+                .buildAsyncClient();
+        
+        System.out.println("Successfully created metrics clients");
+    }
+}
+```
+
+### Query metrics using batch operations
+
+The method signatures for querying metrics in batch operations remain largely the same, but you'll need to update the import statements:
+
+Previous code:
+```java
+import com.azure.monitor.query.MetricsClient;
+import com.azure.monitor.query.MetricsClientBuilder;
+import com.azure.monitor.query.models.MetricsQueryResult;
+import com.azure.monitor.query.models.MetricsQueryResourcesOptions;
+import com.azure.monitor.query.models.QueryTimeInterval;
+import com.azure.core.credential.TokenCredential;
+import java.util.Arrays;
+import java.util.Map;
+import java.time.Duration;
+import java.time.OffsetDateTime;
+
+public class BatchMetricsQuerySample {
```

Comment:
We don't use "Batch" anywhere in our APIs. So, it's best to avoid this term in the migration guide.